frontend/src/features/properties/list/columns.tsx

```diff
@@ -33,6 +41,19 @@ const spacing = {
   xxlarge: unit * 8,
 };
 
+const getProjectLinkNoDrafts = (namespace: string = 'properties') => (cellInfo: any) => {
+  const projectNumbers = _.filter(cellInfo.value, (p: string) => !p.includes('DRAFT'));
```

Comment:
don't include draft projects

Reply:
Is there a reason not to include the draft projects?

Reply:
The story says "only show for approved surplus projects" which I assumed meant no drafts?

Reply:
hmmmm, I doubt that is correct, but Mikaela did put that in as a requirement.  We can get a new story if she wants it different.